target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -545,6 +545,15 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "wan" "WAN" "wan" "eth1"
+	ucidef_set_led_netdev "lan" "LAN" "lan" "eth0"
```

Comment:
wan and lan LEDs are not defined on dts.

Reply:
@DragonBluep should be default ? It works on my device without specifying anything

Reply:
Seems they are hardware LEDs. Please remove them here.

Reply:
Done